### ISSUE TITLE
Any chance of a new NuGet package version?

### ISSUE THREAD

**Issue by maartenba** [duffelhq/duffel-api-dotnet#87]
Hi folks,

Would it be possible to publish a new version of the NuGet package, including latest changes?

Thank you! 🙏

**Comment by SamHutchings** [comment]
Of course! I can release one today. I'll reply on here when I've done so.

**Comment by SamHutchings** [comment]
Available on [Nuget](https://www.nuget.org/packages/Duffel.ApiClient/1.0.0-RC5) now, thanks always for your contributions @maartenba!

**Comment by maartenba** [comment]
Awesome, thanks!
